walkthrough.ipynb

```diff
@@ -256,9 +253,14 @@
    "source": [
     "## Ground Truth for Models\n",
     "\n",
-    "In order to generate ground truth, we need to generate ground truth, or a human-generated baseline that we can compare the model against to see how to good it is. The term ground truth comes from geology and geospatial sciences, where actual information was collected on the ground to validate data acquired through remote sensing, such as satellite imagery or aerial photography. Since then, the concept has been adopted in other fields, particularly in machine learning and artificial intelligence, to refer to the accurate, real-world data used for training and testing models. \n",
+    "The term ground truth comes from geology and geospatial sciences, where actual information was collected on the ground to validate data acquired through remote sensing, such as satellite imagery or aerial photography. Since then, the concept has been adopted in other fields, particularly in machine learning and artificial intelligence, to refer to the accurate, real-world data used for training and testing models. \n",
     "\n",
-    " We'll do this by performing inference against existing models that are trained for summarization. Let's take a look at the data we'll be using first. [Generating GT](https://thunderbird.topicbox.com/groups/addons/T18e84db141355abd-M4cca8e3f9e4fee9ae14b9dbb/self-hosted-version-of-extension-is-incorrectly-appearing-in-atn)\n"
+    "The best ground truth is human-generated but building it is a very expensive task.\n",
```

Comment:
I am not sure whether I maintained your original meaning, please LMK what you think abt it. I wanted to emphasize that even if we do the automatic generation now it's just for the sake of explanation and we'll evaluate keeping that in mind